packages/apps/transfer/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "_phase:build": "npm run build",
     "_phase:test": "npm run test",
-    "build": "next build",
+    "build": "cross-env NODE_ENV=production next build",
```

Comment:
Do we really need this? To my knowledge `next build` automatically sets this (https://github.com/vercel/next.js/discussions/13410#discussioncomment-18760)

Reply:
yep, this will distinguish between dev or prod in next.config file later to pass regex or simple file extension, otherwise it'll always hit the TSX extension and try to build test files which will later throw error during build